android/src/main/kotlin/com/amplitude/amplitude_flutter/AmplitudeFlutterPlugin.kt

```diff
@@ -127,6 +154,31 @@ class AmplitudeFlutterPlugin : FlutterPlugin, MethodCallHandler {
         }
     }
 
+    private fun trackAppLifecycleAndDeepLinkEvents(appLifecycles: Boolean, deepLinks: Boolean) {
+        amplitude.isBuilt.invokeOnCompletion { exception ->
+            if (exception != null) {
+                println("isBuilt computation failed with exception: $exception")
+            } else {
+                val utils = DefaultEventUtils(amplitude)
```

Comment:
> Is this already in the repo? I can't find it.

Hey @justin-fiedler, it's used here.

Reply:
Can you link to the file?

Reply:
You mean this? https://github.com/amplitude/Amplitude-Kotlin/blob/111007ff903c5d1af07a3d5b09160dfb94dd2c86/android/src/main/java/com/amplitude/android/utilities/DefaultEventUtils.kt#L14

Reply:
Ah oops, I thought it was the Fluter repo. 🤦